### PR TITLE
Update plot_timeline script to use the new log format.

### LINKED ISSUE #112: Broken Script Tracking Issue
### Broken Script
Changing the common log format causes the `plot_timeline.py`  script to break because it's using the previous format to parse the `event.log`. Currently, we're in the middle of refactoring the layers to use the `CommonLogger` that writes the events in the new format. We want to finish refactoring all the layers before updating the script. This issue is created to track the broken script and fix it after the refactoring is over.

We can fix this issue when the following layers are refactored:
- - [x] Frame time layer
- - [x] Compile time layer
- - [x] Runtime layer
- - [x] Cache sideload layer

- [x] Update `plot_timeline.py`

### PR DESCRIPTION
**Pull request by miladHakimi**
Updating the event log format causes this script to break because it was using the old format. This PR fixes this issue.

Fixes: #112

### CODE REVIEW

File: scripts/plot_timeline.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/env python3
+#!/ usr / bin / env python3
```
Comment (kuhar) [inline scripts/plot_timeline.py]: this seems broken

File: scripts/plot_timeline.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 # Copyright 2020 Google LLC
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
+# Licensed under the Apache License, Version 2.0(the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     https://www.apache.org/licenses/LICENSE-2.0
+# https:  // www.apache.org/licenses/LICENSE-2.0
```
Comment (kuhar) [inline scripts/plot_timeline.py]: also here

File: scripts/plot_timeline.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/env python3
+#!/ usr / bin / env python3
 # Copyright 2020 Google LLC
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
+# Licensed under the Apache License, Version 2.0(the "License");
```
Comment (kuhar) [inline scripts/plot_timeline.py]: can you double check the license header?

File: scripts/plot_timeline.py [hunk 4]
```diff
@@ -91,48 +137,86 @@ def get_pipeline_creation_times(create_pipeline_events):
...
+
+def parse_event_log(log: str) -> (str, int, str):
+    """
+    Returns a triple containing event name, timestamp, and rest of event which
+    may be empty or contain attributes.
+    log format sample:
+    sample1: `compile_time_layer_init,timestamp:123`
+    sample2: `frame_present,timestamp:1667942408738000395,frame_time:9707270,started:1`
+    """
+# This pattern captures `event_type`, `timestamp`, and the rest of the event.
```
Comment (kuhar) [inline scripts/plot_timeline.py]: this comment is not aligned with the code

File: scripts/plot_timeline.py [hunk 5]
```diff
@@ -91,48 +137,86 @@ def get_pipeline_creation_times(create_pipeline_events):
...
     ax.set_ylabel('Creation Time [ms]')
     max_creation_time_millis = 0
 
     if 'create_graphics_pipelines' in events_by_type:
-        xs, ys = get_pipeline_creation_times(events_by_type['create_graphics_pipelines'])
-        ax.scatter(xs, ys, s=3, color=plt.cm.tab10(8), label='Create Graphics Pipelines')
+        xs, ys = get_pipeline_creation_times(
+            events_by_type['create_graphics_pipelines'])
+        ax.scatter(xs, ys, s=3, color=plt.cm.tab10(
+            8), label='Create Graphics Pipelines')
```
Comment (kuhar) [inline scripts/plot_timeline.py]: I think we should keep these unrelated formatting changes separate. Either undo them here and elsewhere or open a PR to format the code you are going to modify first.

Comment (kuhar) [reply scripts/plot_timeline.py]: Also elsewhere in this file.

**Review by kuhar — CHANGES_REQUESTED** [review]

File: scripts/plot_timeline.py (hunk 5 above)
Comment (miladHakimi) [reply scripts/plot_timeline.py]: Sure!

**Review by kuhar — APPROVED** [review]
